data_format.md

```diff
@@ -109,19 +115,31 @@ SCENE_DTYPE = [
 ```
 
 ### Frames
-A frame consists of information about the ego vehicle (e.g. where it was at that time), a timestamp, and a reference to the agents in that frame. Because there may be multiple agents observed we store the start and the end index.
+A frame captures all information that was observed at a time. This includes
+
+- the timestamp, which the frame describes;
+- data about the ego vehicle itself such as rotation and position;
+- a reference to the other agents (vehicles, cyclists and pedestrians) that were captured by the ego's sensors;
+- a reference to all traffic light faces for all visible lanes.
```

Comment:
I actually do not think traffic light faces is a common English word, so one could wonder what that is upon arriving here - it is solved in the paragraph below, so not sure how important. One could just say 'traffic lights' here or add 'see below' to help the reader here?

Reply:
second this! It was a bit confusing to me at first as well!